qds_sdk/account.py

```diff
@@ -94,7 +94,7 @@ def branding(args):
         return result
 
 class Account(SingletonResource):
-    credentials_rest_entity_path = "accounts/get_creds"
+    credentials_rest_entity_path = "accounts/creds"
```

Comment:
@anumahadevan this fails if user tries v1.2, also tried this with v2 and is failing at this endpoint